File: apzkr-pzpi-21-2-volnykov-kyrylo/Task4-MobileClient/AquaTrackMobile/app/src/main/java/com/example/aquatrackmobile/models/FeedingSchedule/FeedingSchedule.kt
package com.example.aquatrackmobile.models.FeedingSchedule

import kotlinx.serialization.Serializable

@Serializable
data class FeedingScheduleResponse(
    val feedingSchedule: FeedingSchedule
)

@Serializable
data class FeedingSchedule(
    val feedingScheduleId: Int,
    val aquariumId: Int,
    val feedTime: String,
    val feedAmount: String,
    val feedType: String,
    val repeatInterval: String,
    val active: Boolean
)

File: apzkr-pzpi-21-2-volnykov-kyrylo/Task1-Server/AquaTrack/AquaTrack/Models/FeedingSchedule.cs
namespace AquaTrack.Models
{
    public class FeedingSchedule
    {
        public int FeedingScheduleId { get; set; }
        public int AquariumId { get; set; }
        public float FeedTime { get; set; }
        public float? FeedAmount { get; set; }
        public string? FeedType { get; set; }
        public float? RepeatInterval { get; set; }
        public bool Active { get; set; }

        public Aquarium Aquarium { get; set; }
    }
}
